vhdbuilder/packer/packer_source.sh

```diff
@@ -99,6 +99,10 @@ copyPackerFiles() {
   AKS_LOGROTATE_TIMER_DROPIN_DEST=/etc/systemd/system/logrotate.timer.d/override.conf
   AKS_LOGROTATE_CONF_SRC=/home/packer/rsyslog
   AKS_LOGROTATE_CONF_DEST=/etc/logrotate.d/rsyslog
+  AKS_CHECK_NETWORK_SRC=/home/packer/aks-check-network.sh.sh
+  AKS_CHECK_NETWORK_DEST=/usr/local/bin/aks-check-network.sh.sh
```

Comment:
can probably put this in `/opt/azure/containers/aks-check-network.sh` instead

Reply:
also seems `.sh` is repeated twice here?

Reply:
Fixed! thanks for pointing that out! I moved it to `/opt/azure/containers` too.